### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by flore77**
## Need

```gherkin
As a developer
I want to have docs about everything
So that the users don't bother me with questions
```

## Deliverables
- [x] [webdriverIO](https://github.com/uberVU/mugshot/blob/02f6cecc2c48a8a6e70eaf8bd254bf193fa79695/lib/adapters/webdriverio.js) adapter
- [x] looksame adapter [userOptions](https://github.com/uberVU/mugshot/blob/02f6cecc2c48a8a6e70eaf8bd254bf193fa79695/lib/adapters/looks-same.js#L11)
- [x] doc page about how to write your own adapter

## Solution
Write & review

### CODE REVIEW

**Comment by flore77** [comment]
@andrei-picus-hs I have added the docs for the `webdriverio` and `looks-same` adapters. I also have created the wiki page for [Writing  your own adapter](https://github.com/uberVU/mugshot/wiki/Mugshot-Adapters).

The wiki has also created a [Home page](https://github.com/uberVU/mugshot/wiki), I guess it is default. We can populate it with the [README.md](https://github.com/uberVU/mugshot/blob/master/README.md), is that ok ?

I was also thinking it would be great to do a video tutorial, when I write the `casper adapter`. Of course I will write it before, review etc., and then do the tutorial :laughing: Or you can do it yourself if you want, but it would be very cool and useful I think. let me know :smiley:

File: lib/adapters/looks-same.js [hunk 1]
```diff
@@ -7,6 +7,12 @@ var objectAssign = require('object-assign');
 *
 * @implements {Differ}
 * @class
+*
+* @param {Object} [userOptions] - Options for LooksSame
+* @param {Boolean} [userOptions.strict] - Verifies every pixels
```
Comment (andrei-picus-hs) [inline lib/adapters/looks-same.js]: The description is misleading. `looks-same` checks all the pixels but you can control the mismatch tolerance. You should explain that there's a relation between `strict` and `tolerance`, mainly that you can't have both together.

You should also add a link to the `looks-same` docs.

**Comment by andrei-picus-hs** [comment]
> The wiki has also created a Home page, I guess it is default. We can populate it with the README.md, is that ok ?

We don't want to maintain 2 pages with the same information. You can leave it like it is right now or you could add a simple helper text e.g. 'use the links on the right to find out stuff'.

> I was also thinking it would be great to do a video tutorial, when I write the casper adapter
>  it would be very cool and useful I think

Don't know :confused: we can talk about this at some point.

**Comment by flore77** [comment]
I have added 2 phrases on the homepage.

I have turned `exemplification` into `example`.

I have add some more docs to the LooksSame Adapter.

> Don't know :confused: we can talk about this at some point.

Sure, it is just an idea.

File: lib/adapters/looks-same.js [hunk 2]
```diff
@@ -5,8 +5,23 @@ var objectAssign = require('object-assign');
 /**
 * LooksSame Adapter for use with Mugshot
 *
+* It can be used in strict mode that checks every pixel to look the same
+* and a tolerance mode which permits an error range, this two options
+* cannot be used togheter
+*
+* Although if you pass both Mugshot will set strict mode to false
+*
+* [LooksSame options] {@link https://github.com/gemini-testing/looks-same}
```
Comment (andrei-picus-hs) [inline lib/adapters/looks-same.js]: Use this for the `userOptions` param.

**Comment by flore77** [comment]
done. What tool is used to create the docs from the `JSDoc annotations` ? It would be good to create the documentation and to browse it a little bit. This way we can figure out if something is missing.

**Comment by flore77** [comment]
@andrei-picus-hs please review :smile:

**Comment by andrei-picus-hs** [comment]
LGTM

**Pull request merged**